IGHTMLQuery/Classes/IGXMLNodeSet.h

```diff
@@ -51,15 +51,15 @@ typedef void (^IGXMLNodeSetEachBlock)(IGXMLNode* node);
  */
 @interface IGXMLNodeSet : NSObject <NSFastEnumeration, IGXMLNodeSetCore, IGXMLNodeManipulation, IGXMLNodeQuery>
 
--(id) initWithNodes:(NSArray*)nodes;
+-(nullable id) initWithNodes:(nullable NSArray<IGXMLNode*>*)nodes;
```

Comment:
So, this goes back to the above comment on IGXMLNodeQuery.h -- where it's technically possible that the init could fail, but.... it's probably remote that it will fail AND if it does fail, you're probably in a bad situation anyways. So, maybe just make it nonnull (and if we happen to hit the odd chance that it would crash, well... we're likely so far gone anyways that it won't matter, we'd probably be crashing anyways)?